docs/modeling/processes/variables.md

```diff
@@ -81,21 +85,60 @@ The following is an example payload for `POST /v1/process-instances` in the runt
 }
 ```
 
-## Mapping process variables
-Process variables can be mapped to and from variables in a BPMN element so that their values can be reused and updated throughout a process instance. 
+## Mapping variables
+Process variables in a process can be mapped to and from variables in BPMN elements such as [task variables](../forms/README.md#form-variables), [script variables](../scripts.md#script-variables) and [decision table values](../decisions.md). Input mapping is used to set the process variable sent from the process to the BPMN element and output mapping is used to set the target process variable to receive the results from the BPMN element after it has been executed.
+
+The mapping of variables is stored in the `<process-definition-name>-extensions.json` file and can also be viewed in the **Extensions Editor** in the `mappings` section.
+
+Mapping variables requires configuring two fields:
+
+* [Variable type](#variable-type)
```

Comment:
Not sure if `Variable type` is the best option for naming it, because the mapping can be to a variable or to value. I know it is out of my concern but I think that calling this `Mapping type` and the following one as `Mapping mode` would make more sense in my opinion

Reply:
I agree that these are confusing - especially with the option to send values rather than variables. I've updated them to `variable mapping` and `mapping type` to now as I'm a bit constrained by the UI text at the moment.